app/i18n/translations/fr.json

```diff
@@ -936,24 +922,24 @@
   "logs.subtitle": "Journaux Système",
   "mixer.ntf.createdAcct": "Comptes créés avec succès et le mixeur a été configuré.",
   "mixer.ntf.startMixerFailed": "{originalError}",
-  "mixer.start.running": "Le mixeur et/ou l'achat automatisé de tickets sont en cours d'exécution, veuillez les arrêter avant de démarrer le mixeur'.",
-  "mixerSetting.header": "Paramétrages du Mixeur",
+  "mixer.start.running": "Le mixeur et/ou l'achat automatique de tickets sont en cours d'exécution, veuillez les arrêter avant d'acheter un ticket'.",
+  "mixerSetting.header": "Configuration du Mixeur",
   "mixerSettings.gotIt": "J'ai compris",
   "modal.complete.registration.process": "Terminer le processus d'enregistrement",
   "modal.go.to.tickets.status": "Consulter le statut du ticket",
   "myTicket.syncVSP": "Échec de la synchronisation des tickets du VSP",
   "myTickets.loadingMoreTickets": "Chargement d'autres tickets...",
   "myTickets.loadingMoreTicketsProgressAsc": "Bloc ascendant {block} ({blockPerc, number, percent})",
   "myTickets.loadingMoreTicketsProgressDesc": "Bloc descendant {block} ({blockPerc, number, percent})",
-  "myTickets.noloadingMoreTickets": "Chargement de plus de tickets annulé",
+  "myTickets.noloadingMoreTickets": "Chargement de plus de tickets annulés",
```

Comment:
I think I was wrong on this one.
C'est le chargement qui est annulé, on ne parle pas de "tickets annulés"